src/esri-metadata.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash')
+const template = require('./esri-template-metadata')
+
+module.exports = function (req, res) {
+  // Adjust layer specific properties
+  this.model.pull(req, (e, geojson) => {
+    const { metadata } = geojson
```

Comment:
Does Koop guarantee a `metadata` even if the provider does not add one?

Reply:
This is a good question.  No, Koop doesn't guarantee it.  I have updated the code to make it fault tolerant for the case where a provider doesn't add a `metadata` property.